OpenSim/Simulation/BufferedOrientationsReference.h

```diff
@@ -79,19 +79,19 @@ class OSIMSIMULATION_API BufferedOrientationsReference
             SimTK::Array_<SimTK::Rotation_<double>>& values) const override;
 
     /** add passed in values to data procesing Queue */
-    void putValues(double time, const SimTK::RowVector_<SimTK::Rotation>& dataRow);
+    void putValues(double time, const SimTK::RowVector_<SimTK::Rotation_<double>>& dataRow);
```

Comment:
Isn't `SimTK::Rotation` an alias for `SimTK::Rotation_<SimTK::Real>`, with `SimTK::Real` _usually_ being an alias for `double` (i.e. the before and after are equivalent for OpenSim)?